app/konnect/vitals/custom-reports.md

```diff
@@ -0,0 +1,119 @@
+    the time frame selector. For example:
+
+    ```
+    2022-01-27T21:30:58Z to 2022-02-03T21:30:58Z
+    ```
+
+
+1. Choose a report type.
+
+   * **Service report**: Generate a report based on Services.
+   * **Route report**: Generate a report based on Routes.
+   * **Application report**: Generate a report based on API consumers.
+
+   Depending on the report type you choose, the available metrics and entities
+   will change.
+
+1. Choose a [metric](#metrics) to group the data by.
+1. Choose the entities to focus on in your report.
+1. Click **Create**.
+
+The report details page opens. Here you can:
+
+* **View details**: Hover over a bar on the graph to see details about the data.
+* **Filter the graph**: Click on an item in the legend to temporarily hide it from view,
+and click it again to show the item. The graph resets on a refresh.
+
+## Export a custom report
+
+You can export any custom report in CSV format.
+
+1. From the left-side menu in Konnect, open ![](/assets/images/icons/konnect/icn-vitals.svg){:.inline .no-image-expand}
+**Vitals**, then **Reports**.
+1. Click on a report row to open the report's page.
+1. Click the **Actions** dropdown and select **Export Report**.
+
+  Konnect generates a CSV file download with all the data in the report.
+
+## Edit a custom report
+
+You can edit a report in one of the following ways:
+
+1. From the left-side menu in Konnect, open ![](/assets/images/icons/konnect/icn-vitals.svg){:.inline .no-image-expand}
+**Vitals**, then **Reports**.
+1. Either:
+   * Click the ![](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
+   settings icon for a report row and select **Edit**.
+   * Click on a report row to open the report. In the **Actions** dropdown menu,
+   select **Edit Report**.
+1. Make your changes and click **Update**.
+
+  The report page opens. You may have to refresh the page to see the updated
+  report.
+
+## Delete a custom report
+
+You can delete a report in one of the following ways:
+
+1. From the left-side menu in Konnect, open ![](/assets/images/icons/konnect/icn-vitals.svg){:.inline .no-image-expand}
+**Vitals**, then **Reports**.
+1. Either:
+   * Click the ![](/assets/images/icons/konnect/konnect-settings.svg){:.inline .no-image-expand}
+   settings icon for a report row and select **Delete**.
+   * Click on a report row to open the report. In the **Actions** dropdown menu,
+   select **Delete Report**.
+1. Confirm deletion in the dialog.
+
+## Metrics
+
+Each metric depends on a time frame and a primary entity (report type).
+
+Metric | Report type | Description
+-------|------------|------------
+Total traffic | Service, Route, Application | Total number of API calls within the selected time frame.
+Total traffic by status code | Service, Route, Application | Number of API calls grouped by status code.
+Total traffic by service | Route, Application | Number of API calls grouped by Service(s) or Service versions.
```

Comment:
minor: I believe this is always grouped by Service. Never service version.

At least as of today. In the coming months, we expect to add the ability to see a report grouped by service version.

It's a bit, imo, confusing as we have it today. On the create and edit pages, you can pick to filter by service or service version, but then on the reports detail page, it will only show service. EVEN IF you only selected a single service version. So if you have service Foo with Foo0.1 and Foo0.2 service versions and you select only Foo0.1 and then save your report, the report detail page shows you a report for Foo with just the subset of the traffic for Foo0.1.

To say it another way: I would say that we let you filter by either service or service version, but the group by is always by service.